src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/AbstractLSPInlayProvider.java

```diff
@@ -69,9 +72,23 @@ public ImmediateConfigurable createConfigurable(@NotNull NoSettings o) {
             @NotNull
             @Override
             public JComponent createComponent(@NotNull ChangeListener changeListener) {
-                return LayoutKt.panel(new LCFlags[0], "LSP", builder -> {
-                    return null;
-                });
+                JLabel label = new JLabel();
```

Comment:
This is displayed in the settings: Editor > Inspections > LSP
The code seems incorrect because the settings report "Can't find inspection description".
![image](https://user-images.githubusercontent.com/25126/227900378-32439b58-672b-4547-9dd3-9afbe3119679.png)
This PR should not correct this (solution requires some digging, is not obvious) but provide compatibility.